src/components/StatsScoringDialog.vue

```diff
@@ -17,73 +17,66 @@
           variant="text"
           data-cy="ranked-info-button"
         >
-          <span v-if="showButtonText">How are ranks determined?</span>
+          <span v-if="showButtonText">{{ t('game.dialogs.statsScoringDialog.ranksDetermination') }}</span>
```

Comment:
@arnaut08 let's rename this string to `rankDetermination`
```suggestion
          <span v-if="showButtonText">{{ t('game.dialogs.statsScoringDialog.rankDetermination') }}</span>
```